packages/react/src/isomorphicClerk.ts

```diff
@@ -166,7 +166,10 @@ export default class IsomorphicClerk {
         await global.Clerk.load(this.options);
       }
 
-      return global.Clerk?.loaded ? this.hydrateClerkJS(global.Clerk) : undefined;
+      if (global.Clerk?.loaded || global.Clerk?.isReady()) {
```

Comment:
❓Aren't both returning the same value ? (this.#isReady)

Reply:
Yup, but Clerk.loaded does not exist on clerk-js@3 and we'd like clerk-react to be able to load either both Clerk v3 and v4